### PR TITLE
Create gripsecurity.json

### PR DESCRIPTION
**Pull request by sounil**
Note the addition of "dataTypes", "dataClassification", and "assetOwner"

### CODE REVIEW

File: vendors/gripsecurity.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  "dataClassification": "internal",
+  "aliases": "",
+  "cyberDefenseMatrixMap": {
+    "assetOwner": ["third-party"],
+    "devices": [],
+    "applications": ["identify", "protect"],
+    "networks": [],
+    "data": [],
+    "users": []
+  }
```
Comment (erkangz) [inline vendors/gripsecurity.json:21]: how about we simplify this to: 

```
"assetOwner": "third-party",
"assetCategory": ["application"],
"controlDomain": ["identify", "protect"]
```

This makes it easier to insert these properties to graph entities. Graph entities do not support nested properties.

**Review by erkangz — CHANGES_REQUESTED** [review]

File: vendors/gripsecurity.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  "productCapabilities": ["saas discovery", "saas access management", "saas security"],
+  "description": "Grip Security provides visibility, governance, and data security to help organizations secure their SaaS ecosystem.",
+  "iconWebLink": "",
+  "webLink": "https://www.grip.security/",
+  "website": "https://www.grip.security/",
+  "linkToDPA": "https://www.grip.security/privacy-policy",
+  "linkToSLA": "https://www.grip.security/terms-of-service",
+  "_type": "grip_security",
+  "dataType": ["email", "oidc_logs"],
+  "dataClassification": "internal",
```
[the remark this answers is not in the document]
Comment (erkangz) [reply vendors/gripsecurity.json:12]: also - should we define an enumeration of data types, or is it free form?

File: vendors/gripsecurity.json [hunk 3]
```diff
@@ -0,0 +1,22 @@
+{
+  "name": "Grip Security",
+  "productCapabilities": ["saas discovery", "saas access management", "saas security"],
+  "description": "Grip Security provides visibility, governance, and data security to help organizations secure their SaaS ecosystem.",
+  "iconWebLink": "",
```
Comment (erkangz) [inline vendors/gripsecurity.json:5]: Can we find its icon?

File: vendors/gripsecurity.json (hunk 1 above)
Comment (erichs) [reply vendors/gripsecurity.json:21]: > Graph entities do not support nested properties.

Comment (erichs) [reply vendors/gripsecurity.json]: schema updated via https://github.com/JupiterOne/vendor-stack/pull/33

**Review by erichs — APPROVED** [review]

**Pull request merged**